modules/client-handler/src/integrationTest/java/org/apache/ignite/client/handler/TestServer.java

```diff
@@ -84,6 +90,9 @@ public TestServer() {
                 generator,
                 new TestConfigurationValidator()
         );
+        this.connectorConfiguration = connectorConfiguration == null
+                ? configurationManager.configurationRegistry().getConfiguration(ClientConnectorConfiguration.KEY)
```

Comment:
Do we really need this? Can we always pass a non-null configuration and get rid of the `configurationManager`?